hogben/tests/test_fisher.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import pytest
+
+from hogben.simulate import simulate
+from hogben.utils import fisher
+from refnx.reflect import SLD
+from unittest.mock import patch
+
+
+@pytest.fixture
+def model_fixture():
```

Comment:
call this `refnx_model` or similar, you have already made it clear that it is a fixture